include/filehandler/pemhandler.h

```diff
@@ -13,22 +13,23 @@ namespace ih
 class PemFileReader : public IFileHandler
 {
 public:
-    explicit PemFileReader(std::string const filePath);
-
-    bool isFileValid() const override;
+    explicit PemFileReader(std::string const &filePath);
 
     Address getAddress() const;
 
     bytes getSeed() const;
 
     bytes getPrivateKey() const;
 
+protected:
+    bool isFileValid() const override;
+
 private:
     std::string getFileContent() const;
 
-    bytes getKeyBytesFromFile() const;
+    bytes getKeyBytesFromContent(std::string const &content) const;
```

Comment:
Regarding method: `getKeyBytesFromContent` I somehow disagree. This method **does not read** anything from any file. It gets as input the content of a file, base64 decodes it, and returns a byte array from it.

However, one might argue about `getFileContent` method's name, since this one actually reads the file content's. The only doubt I have about renaming it is because when you see `readContent`, the name doesn't necessarily points that it returns something, but rather it should have an in-out paramter (e.g.: `readContent(std::string& content)`).